example/self-test/base.yaml

```
@@ -0,0 +1,23 @@
+      labels:
+        app: base
+    spec:
+      containers:
+        - name: base
+          image: serverless-registry.cn-shanghai.cr.aliyuncs.com/opensource/test/test-base:1.3.4 # 已经打包好的镜像
+          imagePullPolicy: Always
+          ports:
+            - name: base
+              containerPort: 8080
+            - name: arklet
+              containerPort: 1238
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)